app/pages/multisig/ProposalView/util.ts

```diff
@@ -144,8 +150,31 @@ async function HandleUpdateInstructionSignatureFile(
         };
     }
 
-    let validSignature = false;
-    validSignature = await isSignatureValid(proposal, signature);
+    let blockSummary;
+    try {
+        blockSummary = (await fetchLastFinalizedBlockSummary())
+            .lastFinalizedBlockSummary;
+    } catch {
+        return {
+            show: true,
+            header: errorMessages.unableToReachNode,
+            content:
+                'Unable to verify signature is signed by an authorized key without connection to node.',
```

Comment:
Unable to verify that the signature is signed by an authorization key without a connection to a node.